src/lib/components/media/riven/item-manual-scrape.svelte

```
@@ -0,0 +1,891 @@
+            fetchSettings();
+        }
+    });
+
+    async function handleAutoScrape() {
+        // itemId is optional now, as we can fallback to externalId
+        
+        loading = true;
+        error = null;
+
+        try {
+            const body: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)